misk-hibernate/src/test/kotlin/misk/hibernate/actions/HibernateDatabaseQueryMetadataFactoryTest.kt

```diff
@@ -0,0 +1,160 @@
+package misk.hibernate.actions
+
+import misk.hibernate.Constraint
+import misk.hibernate.DbRoot
+import misk.hibernate.DbTimestampedEntity
+import misk.hibernate.Id
+import misk.hibernate.Operator
+import misk.hibernate.Order
+import misk.hibernate.Projection
+import misk.hibernate.Property
+import misk.hibernate.Query
+import misk.hibernate.Select
+import misk.hibernate.Session
+import misk.hibernate.actions.HibernateDatabaseQueryStaticAction.Companion.HIBERNATE_QUERY_STATIC_WEBACTION_PATH
+import misk.web.metadata.DatabaseQueryMetadata
+import misk.web.metadata.Field
+import misk.web.metadata.Type
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.time.Instant
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Table
+
+class HibernateDatabaseQueryMetadataFactoryTest {
+  val converter = HibernateDatabaseQueryMetadataFactory(
+      listOf()
+  )
+
+  @Test
+  fun happy() {
```

Comment:
if I understand correctly, this test the static query right? maybe we include one for dynamic query metadata as well?

Reply:
Good call, will do!